src/main/kotlin/no/nav/helse/spade/behandlinger/BehandlingDto.kt

```diff
@@ -0,0 +1,7 @@
+package no.nav.helse.spade.behandlinger
+
+data class BehandlingDto(val behandlingsId: String,
```

Comment:
`BehandlingHeader` eller et annet fornuftig navn som sier noe om at denne klassen representerer et lite utvalg av felter for en behandling?

Reply:
`BetalingSummary`, kanskje, selv om det ikke er så mye oppsummerende informasjon...

Reply:
Slet med å finne navn sjæl. `BehandlingSummary` kan vi bruke

Reply:
Ofte vanskelig!